config/locales/en/providers.yml

```diff
@@ -679,6 +679,8 @@ en:
           policy_disregards: Discretionary disregards selected
           multi_benefit: Multiple benefits in single bank transaction
           dwp_override: Provider disagrees with DWP result
+          restrictions: Restrictions on client's assets
+          further_employment_details: Further employment details
```

Comment:
For my benefit, what do we do about adding welsh translations?

Reply:
@jsugarman we have a rake task cy.rake that generates the dummy backwards welsh translations. However, I believe this only covers the citizens and not the providers journey (i.e there is no locales/cy/providers.yml file).